decoders/connector/milesight/ct101/description.md

```diff
@@ -0,0 +1 @@
+The Milesight CT101 LoRaWAN Sensor is a unique current monitoring solution designed for seamless integration into existing electrical systems. This self-powered sensor eliminates the need for batteries, offering long-term operation and reduced maintenance.
```

Comment:
Make this smaller, don't forget that this shows up here:

![image](https://github.com/user-attachments/assets/4267e705-6bde-4b79-ad13-efbf8a7965ad)